datadotworld/client/api.py

```diff
@@ -704,11 +714,44 @@ def download_file(self, dataset_key, file):
         except _swagger.rest.ApiException as e:
             raise RestApiError(cause=e)
 
+    # Streams Operation
+
+    def append_records(self, dataset_key, stream_id, body,
+                       desired_mimetype='application/json', **kwargs):
```

Comment:
It would be clearer if we called this `provided_mimetype` as opposed to `desired_mimetype`.